roles/libvirt_manager/molecule/check_dns/converge.yml

```diff
@@ -19,7 +19,7 @@
   gather_facts: true
   vars:
     ansible_user_dir: "{{ lookup('env', 'HOME') }}"
-    cifmw_basedir: "/opt/basedir"
```

Comment:
that shouldn't be part of the current change.

Reply:
(this leads to the removal of all the logs - you really don't want that, do you?)